src/client/utils/styles/_tiles-item.scss

```diff
@@ -15,6 +15,8 @@
  * limitations under the License.
  */
 
+@import 'browser-type';
```

Comment:
Hmm, don't you need ` @import '../../imports';` here? `@ellipsis` is used there also ...